custom/enikshay/integrations/nikshay/tests/test_repeaters.py

```diff
@@ -600,6 +600,7 @@ def test_payload_properties(self, _):
         self.assertEqual(payload['Outcome'], '7')
 
 
+@use_sql_backend
```

Comment:
seems like this is a poorly named decorator. What you really want is: `@override_settings(TESTS_SHOULD_USE_SQL_BACKEND=True)`